frontend/src/Components/Header/Header.js

```diff
@@ -1,32 +1,44 @@
 import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
-import { ReactComponent as ArrowIcon } from '../../assets/icons/arrowicon.svg'
-import { ReactComponent as EditIcon } from '../../assets/icons/editicon.svg'
+import { ReactComponent as ArrowIcon } from '../../Assets/icons/arrowicon.svg'
+import { ReactComponent as EditIcon } from '../../Assets/icons/editicon.svg'
+import { ReactComponent as LogoutIcon } from '../../Assets/icons/logouticon.svg'
 
 export default function Header({
   className,
   children,
   secondaryStyle,
   isDetailPage,
+  isPlantList,
   onClickArrow,
   onClickEdit,
+  onClickLogout,
 }) {
   Header.propTypes = {
     className: PropTypes.string,
     children: PropTypes.string.isRequired,
     secondaryStyle: PropTypes.bool,
     isDetailPage: PropTypes.bool,
+    isPlantList: PropTypes.bool,
     onClickArrow: PropTypes.func,
     onClickEdit: PropTypes.func,
+    onClickLogout: PropTypes.func,
   }
 
   return (
     <HeaderStyled className={className}>
       <HeadingContainer secondaryStyle={secondaryStyle}>
-        {isDetailPage && <ArrowIcon onClick={onClickArrow} />}
+        {isDetailPage && (
+          <ArrowIcon onClick={onClickArrow} data-testid="arrow-icon" />
```

Comment:
Perhaps it would be better if you wrap the icon in an (invinsible) button. So that the click area is larger than just the path of the svg.